docs/index.rst

```diff
@@ -1,255 +1,50 @@
-upload sets being used by multiple apps at once. You use the
-`configure_uploads` function to load the configuration for the `UploadSet`s.
-You pass in the app and all of the `UploadSet`s you want configured. Calling
-`configure_uploads` more than once is safe. ::
-
-    configure_uploads(app, (photos, media))
-
-If your app has a factory function, that is a good place to call this
-function.
-
-
-File Upload Forms
-=================
-To actually upload the files, you need to properly set up your form. A form
-that uploads files needs to have its method set to POST and its enctype
-set to ``multipart/form-data``. If it's set to GET, it won't work at all, and
-if you don't set the enctype, only the filename will be transferred.
-
-The field itself should be an ``<input type=file>``.
-
-.. code-block:: html+jinja
-
-    <form method=POST enctype=multipart/form-data action="{{ url_for('upload') }}">
-        ...
-        <input type=file name=photo>
-        ...
-    </form>
-
-
-API Documentation
-=================
-This documentation is generated directly from the source code.
-
-
-Upload Sets
------------
-.. autoclass:: UploadSet
-   :members:
-
-.. autoclass:: UploadConfiguration
-
-
-Application Setup
------------------
-.. autofunction:: configure_uploads
-
-
-Extension Constants
--------------------
-These are some default sets of extensions you can pass to the `UploadSet`
-constructor.
-
-.. autoclass:: AllExcept
-
-.. autodata:: DEFAULTS
-
-.. autodata:: ALL
-
-.. autodata:: TEXT
-
-.. autodata:: IMAGES
-
-.. autodata:: AUDIO
-
-.. autodata:: DOCUMENTS
-
-.. autodata:: DATA
-
-.. autodata:: SCRIPTS
-
-.. autodata:: ARCHIVES
 
-.. autodata:: EXECUTABLES
+Notes on this package
+---------------------
 
+This is an independently maintained version of `Flask-Uploads`
+including four years of unreleased changes, at least not released to PyPI.
 
-Testing Utilities
------------------
-.. autoclass:: TestingFileStorage
+Noteworthy is the fix for the `Werkzeug` API change.
+Please see the migration guide from `flask-Uploads`
 
 
-Backwards Compatibility
-=======================
+Goals
+-----
 
-Version 1.0
------------
-* Removal of `patch_request_class`
-* `autoserve` is now deactivated by default
+- provide a stable drop-in replacement for `Flask-Uploads`.
+- regain momentum for this widely used package.
+- provide working PyPI packages.
 
 
-Version 0.1.3
--------------
-* The `_uploads` module/blueprint will not be registered if it is not needed
-  to serve uploads.
+Contents
+--------
 
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+   :caption: Contents:
 
-Version 0.1.1
--------------
-* `patch_request_class` now changes `max_content_length` instead of
-  `max_form_memory_size`.
+   self
+   Installation <installation>
+   Getting started <getting_started>
+   Configuration <configuration>
+   minimal Application <minimal_app>
+   explanation <explanation>   
+   api <api>
+   versions <versions>
+   contributing <contributing>
```

Comment:
Please use canonical capitalization rules.

Also, here it would be a good place to link to the `CONTRIBUTING.rst` in the top level source directory.